src/bokeh/models/plots.py

```diff
@@ -880,11 +880,11 @@ def __init__(self, *args, **kwargs) -> None:
     index and optional row and column spans (the defaul span is 1).
     """)
 
-    rows = Nullable(TracksSizing, default="max-content", help="""
+    rows = Nullable(TracksSizing, default=None, help="""
     Describes how the grid should maintain its rows' heights.
     """)
 
-    cols = Nullable(TracksSizing, default="max-content", help="""
+    cols = Nullable(TracksSizing, default=None, help="""
     Describes how the grid should maintain its columns' widths.
     """)
```

Comment:
Curious, what are some possible values for these properties? The descriptions I find scarce of meaning.
https://github.com/bokeh/bokeh/blob/fbba86223a7d4c8415e16319fe3c930ee91b3c33/src/bokeh/models/layouts.py#L358

Reply:
I updated the types to be more specific and match bokehjs' types, and added more useful docstrings.